src/platform/nxp/k32w/k32w1/BUILD.gn

```diff
@@ -106,4 +113,7 @@ static_library("k32w1") {
   }
 
   public_deps += [ "${chip_root}/src/crypto" ]
+
+  public_configs =
+      [ "${chip_root}/src/lib/address_resolve:default_address_resolve_config" ]
```

Comment:
this is not ok either - a platform cannot depend on address resolve, because address resolve will eventually depend on platform - platform code is one of the lowest layers we have.

I believe the problem may stem from changes I see in OTA code here - that code is in a `platform` folder however it includes code from `app` which seems a layering inversion. If you do need this (we had cases when platform needed to notify app of wifi changes for example) you have to break that dependency with delegates/callbacks or some other methods. Platform cannot call into app code directly.

Reply:
I've removed the problematic code.